src/event-bridge/event-bus.ts

```diff
@@ -349,27 +437,95 @@ export class EventBus<E extends EventBusRuleInput> extends EventBusBase<E> {
    * new functionless.EventBus.fromBus(awsBus);
    * ```
    */
-  static default<E extends EventBusRuleInput>(stack: Stack): IEventBus<E>;
-  static default<E extends EventBusRuleInput>(scope: Construct): IEventBus<E>;
-  static default<E extends EventBusRuleInput>(
+  static default<E extends EventBusEvent>(stack: Stack): DefaultEventBus<E>;
+  static default<E extends EventBusEvent>(scope: Construct): DefaultEventBus<E>;
+  static default<E extends EventBusEvent>(
     scope: Construct | Stack
-  ): IEventBus<E> {
+  ): DefaultEventBus<E> {
+    return new DefaultEventBus<E>(scope);
+  }
+
+  /**
+   * Creates a schedule based event bus rule on the default bus.
+   *
+   * Always sends the {@link ScheduledEvent} event.
+   *
+   * ```ts
+   * const bus = EventBus.default(scope);
+   * // every hour
+   * const everyHour = bus.schedule(scope, 'cron', aws_events.Schedule.rate(Duration.hours(1)));
```

Comment:
This looks wrong. schedule is static

Reply:
Had the docs backwards between the two implementations. Fixed